src/sentry/stacktraces/processing.py

```diff
@@ -192,16 +195,15 @@ def find_stacktraces_in_data(
     rv = []
 
     def _append_stacktrace(stacktrace: Any, container: Any, is_exception: bool = False) -> None:
-        if not is_exception and (not stacktrace or not get_path(stacktrace, "frames", filter=True)):
+        frames = get_path(stacktrace, "frames", filter=True, default=())
+        if not is_exception and (not stacktrace or not frames):
             return
 
-        platforms = {
-            frame.get("platform") or data.get("platform")
-            for frame in get_path(stacktrace, "frames", filter=True, default=())
-        }
+        platforms = _get_frames_metadata(frames, data.get("platform", "unknown"))
```

Comment:
This is in preparation of my next PR.

Reply:
So this function will eventually get more than just the platforms?

Reply:
Correct.